CMakeLists.txt

```diff
@@ -183,6 +187,9 @@ add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/lib)
 # Bring in our tools folder
 add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/tools)
 
+# Bring in our api folder
+add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/api)
```

Comment:
I agree with Romaric that a new directory is not desirable.

Reply:
Ack.